tests/integration/kafka_test.go

```diff
@@ -331,38 +331,42 @@ func TestKafkaIntegration_bootstrap_inventory(t *testing.T) {
 	}
 }
 
-func TestKafkaIntegration_consumer_offset(t *testing.T) {
-	bootstrapDiscoverConfigInventory := func(command []string) []string {
+func TestKafkaIntegration_producer_test(t *testing.T) {
```

Comment:
Nothing actually changed I've just changed the order of two really similar tests